CHANGELOG.rst

```diff
@@ -1,6 +1,11 @@
-0.4.0 (2019-12-03)
+0.4.1 (2019-12-12)
```

Comment:
As discussed, please change this to `0.5.0`

Reply:
Fixed